### DISCUSSION TITLE
[View Mode] part remove from  Topology dashboard  in v0.8.1?

### DISCUSSION THREAD

**Discussion by yanhongchang** [harmonycloud/kindling#578] [grafana dashbord]
can not found the  [View Mode] in Topology dashboard , is it was remove in v0.8.1?

0.8.0
![image](https://github.com/KindlingProject/kindling/assets/10663031/142086d1-a458-4ae9-8e64-bcbba8be9e73)

0.8.1
![image](https://github.com/KindlingProject/kindling/assets/10663031/5e6a9357-9438-4889-9ea3-977c870249d7)

**Comment by dxsup** [comment]
It's a bug. Thanks for reporting.

**Comment by dxsup** [comment]
Have been fixed in #579.
